ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -35,6 +35,10 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-8-golang-1.19-openshift-4.13
+  ocp_os:
+    name: "4.13"
+    namespace: ocp
+    tag: rhel-coreos-8
```

Comment:
should this tag be rhel-coreos-9 ? asking this because we are later on in Dockerfile we are referencing rhel-coreos-9 tag while pulling osimage from 4.13

Reply:
That's a good point, I'll fix that.